test/new-e2e/tests/agent-metric-logs/linux-log-agent/file_tailing_utils_test.go

```diff
@@ -0,0 +1,109 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package logagent
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	fi "github.com/DataDog/datadog-agent/test/fakeintake/client"
+	componentos "github.com/DataDog/test-infra-definitions/components/os"
+	"github.com/stretchr/testify/assert"
+)
+
+// generateLog generates and verifies log contents.
+func generateLog(s *LinuxVMFakeintakeSuite, content string) {
+	// Determine the OS and set the appropriate log path and command.
+	var logPath, cmd, checkCmd string
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
Recommend tagging these 2 functions as `t.Helper`, it helps readability when debugging errors, as it reports them as their first test caller
Here is an example https://goplay.tools/snippet/QI2B4A6vDvN